src/main/java/com/agilethought/atsceapi/dto/NewHistoryResponse.java

```diff
@@ -0,0 +1,14 @@
+package com.agilethought.atsceapi.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.data.annotation.Id;
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class NewHistoryResponse {
+    @Id
```

Comment:
This Annotation it not required, because this model is a DTO. `@Id` it's for DAO purposes

Reply:
Change done, I have deleted that annotation.